core/geometry/src/test/curve/CurveChainWithDistanceIndex.test.ts

```diff
@@ -43,4 +43,60 @@ describe("CurveChainWithDistanceIndex", () => {
     GeometryCoreTestIO.saveGeometry(allGeometry, "CurveChainWithDistanceIndex", "ClosestPointProblem");
     expect(ck.getNumErrors()).equals(0);
   });
-});
+  const pathWithBSplineProblemFile = "./src/test/testInputs/CurveChainWithDistanceIndex/WoodfieldPath/pathWithBsplines.imjs";
+
+    it("PathWithBsplineLength", () => {
+      const ck = new Checker();
+      const allGeometry: GeometryQuery[] = [];
+
+      const stringData = fs.readFileSync(pathWithBSplineProblemFile, "utf8");
+      if (stringData) {
+        const pathObject = IModelJson.Reader.parse(JSON.parse(stringData));
+        if (pathObject) {
```

Comment:
Wouldn't it be better to `expect` or `assert` on those? It if ever happens, it'll simply skip the test silently.

Reply:
Yes indeed! I've fixed. Thanks!